common/changes/office-ui-fabric-react/master_2018-10-22-23-24.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "office-ui-fabric-react",
-      "comment": "Adding warning and updating documentation on Pivot children needing to be of type PivotItem to be rendered.",
+      "comment": "Removing some of the deprecated components flagged when turning the tslint deprecation flag to true in the office-ui-fabric-react package.",
```

Comment:
Oh I see where the conflict is now. You can just delete this file since it's redundant with the one above.

Reply:
(err, revert changes I mean, not delete)